SU-SWS/stanford_paragraph_types
# Fixed icon paragraph when using images. also improved scss

# READY FOR REVIEW

# Summary
- Added missing display mode for icon grid paragraph type.
- Improved scss and use the colors style sheet.

# Needed By (Date)
- when you can.

# Urgency
- low

# Steps to Test

1. Checkout branch
2. enable/revert stanford_paragraph_types
3. create a story page using an icon grid paragraph item
4. select icon or image and populate appropriately
5. verify colors still look right and the uploaded image displays as expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
